src/main/java/org/identifiers/cloud/ws/linkchecker/daemons/LinkChecker.java

```diff
@@ -32,8 +33,14 @@
  */
 @Component
 public class LinkChecker extends Thread {
-    private static final int WAIT_TIME_LIMIT_SECONDS = 30;
-    private static final int WAIT_TIME_POLL_LINK_CHECK_REQUEST_QUEUE_SECONDS = 3;
+
+    @Value("${org.identifiers.cloud.ws.linkchecker.daemon.linkchecker.waittime.max}")
+    private int wait_time_limit_seconds;
```

Comment:
These should be refactored to match Java's camelCase convention - eg. `private int waitTimeLimitSeconds;` and so on.

Reply:
True, nice catch. I always mix these. Bad habit.